service/http/dffml_service_http/routes.py

```diff
@@ -196,15 +292,45 @@ async def multicomm_dataflow(self, config, request):
                 results = {
                     str(ctx): result async for ctx, result in octx.run(*inputs)
                 }
-                # TODO Implement input and presentation stages?
-                """
-                if config.presentation == "blob":
-                    return web.Response(body=results)
-                elif config.presentation == "text":
-                    return web.Response(text=results)
+                if config.output_mode == "json":
+                    return web.json_response(results)
+
+                # content_info is a list in case of stream
+                # and string in others
+                postprocess_mode, *content_info = config.output_mode.split(":")
+
+                if postprocess_mode == "stream":
```

Comment:
Let's hold off on "stream" output mode for now (just `raise NotImplementedError`). I think we had a miscommunication on this and it'll be best to just fix it after we merge since this is so close to finished.